### PR TITLE
Fixing PR #547, fixed merge conflicts and updated mission hero 'Grow Open Source'

### PR DESCRIPTION
**Pull request by cd10012**
'Push Open Source' => 'Grow Open Source'

Fixes: https://github.com/gitcoinco/web/issues/238

##### Description
This addresses the comments made in the prior pull request that had merge conflicts. Not too sure what was wrong with the branchso I just made a new one. Sorry for the bad form :/ 

##### Checklist

- [x] python3 manage.py runserver 0.0.0.0:8080 is running now
- [x] Reduce size of grow_open_source.png
- [x] Updated Mission Hero
- [x] Addressed other instances of Push open source that the prev pr missed
- [x] commit message follows [commit guidelines] - hope I did it right this time

##### Affected core subsystem(s)
UI on the retail side as well as dashboard. Updated all the copy and image sources, no style or HTML structure changes made.

##### Refers/Fixes
Old PR:https://github.com/gitcoinco/web/pull/547
Issue:   https://github.com/gitcoinco/web/issues/238

### CODE REVIEW

**Comment by mbeacom** [comment]
@cd10012 Thank you for the contribution! The only place I'm still seeing `push open source forward` is at the end of https://github.com/cd10012/web/blob/58bb8c01fcf0e543861e0f8bf5e07adbdcacd3c3/docs/CONTRIBUTING.md `Lets push Open Source Forward.`

**Comment by cd10012** [comment]
Updated, forgot to stage that file!

**Comment by mbeacom** [comment]
Thanks @cd10012 !

**Comment by mbeacom** [comment]
Thoughts on this design being displayed in the `About` section..?

screenshot 2018-03-06 15 56 12

@owocki @PixelantDesign

**Comment by owocki** [comment]
IMHO this image ( https://github.com/gitcoinco/creative/blob/master/freelancers/tshirt/1/art_dark.png ) is a better fit but i broadly defer to @PixelantDesign

**Comment by mkosowsk** [comment]
@mbeacom I like the image but the font is difficult for me to read against the background. I wonder how it would look with a bit of a darker border on the font 🤔

**Comment by PixelantDesign** [comment]
I agree that it’s difficult to read the logo and logotype. I recommend moving the logo below the tree to maintain brand integrity.

**Comment by cd10012** [comment]
@mkosowsk @PixelantDesign I'm pretty bad at photoshop so I'm having trouble moving the logo, but this is how it would look with a darker font color 
dark_font_test

**Comment by mkosowsk** [comment]
@cd10012 I'm not a designer but that looks siiick to me... awesome work! One mega 👍 from me

**Comment by owocki** [comment]
i just looked at the layers of the tree PSD that the designer gave us ( http://bits.owocki.com/0D3d1R1E2G1z/tree.psd ) and it doesnt look like its layered correctly to move around the logo unfortunately...

i wonder if we should just use this tree instead
art_dark

**Comment by cd10012** [comment]
I really like how that one looks as well if my vote counts ;)

**Comment by owocki** [comment]
lets move to the robotic tree then and get this shipped @cd10012 !

**Comment by cd10012** [comment]
@owocki I just gave it a once over should be good to go!

File: app/retail/views.py [hunk 1]
```diff
@@ -63,8 +63,8 @@ def mission(request):
         'active': 'mission',
         'title': 'Mission',
         'card_title': 'Gitcoin is a mission-driven organization.',
-        'card_desc': 'Our mission is to push open source forward.',
-        'avatar_url': '/static/v2/images/mission/hero.png',
+        'card_desc': 'Our mission is to grow open source.',
+        'avatar_url': '/static/v2/images/grow_open_source.png',
```
Comment (owocki) [inline app/retail/views.py]: mind downsizing this image? its HUGE

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -132,4 +132,4 @@ $ git rebase upstream/master
 ```
 
 <img src='https://d3vv6lp55qjaqc.cloudfront.net/items/263e3q1M2Y2r3L1X3c2y/helmet.png'/>
-Welcome to the gitcoin community. Lets push Open Source Forward.
+Welcome to the gitcoin community. Lets Grow Open Source software.
```
Comment (owocki) [inline docs/CONTRIBUTING.md]: Capital S in software pls

Comment (cd10012) [reply docs/CONTRIBUTING.md]: Gotcha! Also the image should be 562kb now.

File: app/tdi/views.py [hunk 3]
```diff
@@ -108,7 +108,7 @@ def whitepaper_access(request, ratelimited=False):
 
     context = {
         'active': 'whitepaper',
-        'title': 'Whitepaper',
+        ' title': 'Whitepaper',
```
Comment (mbeacom) [inline app/tdi/views.py]: Added space

Comment (cd10012) [reply app/tdi/views.py]: Done!

**Comment by owocki** [comment]
merging now.. thanks!

**Comment by owocki** [comment]
6dbca9c4985c09b0d29d5fe4fcb0ff45197b7e96

**Pull request closed without merging**